<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1507</width>
    <height>986</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="accessibleName">
   <string extracomment="méthode des éléments finis avec contraintes d'aire"/>
  </property>
  <property name="documentMode">
   <bool>false</bool>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QWidget" name="widget" native="true">
      <property name="maximumSize">
       <size>
        <width>200</width>
        <height>16777215</height>
       </size>
      </property>
      <layout class="QVBoxLayout" name="verticalLayout">
       <item>
        <widget class="QGroupBox" name="groupBox_2">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>Initialisation</string>
         </property>
         <widget class="QLabel" name="label_2">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>20</y>
            <width>70</width>
            <height>20</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Nb largeur :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_3">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>70</width>
            <height>20</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Nb hauteur :</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox_initialisation_width">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>20</y>
            <width>50</width>
            <height>20</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox_initialisation_height">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>50</y>
            <width>50</width>
            <height>20</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="value">
           <number>5</number>
          </property>
         </widget>
         <widget class="QPushButton" name="Tissue_Button">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>70</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>générer tissu</string>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="pushButton">
         <property name="maximumSize">
          <size>
           <width>150</width>
           <height>16777215</height>
          </size>
         </property>
         <property name="text">
          <string>MEF sans contrainte d'aire</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>100</height>
          </size>
         </property>
         <property name="title">
          <string>MEF avec contrainte d'aire</string>
         </property>
         <widget class="QPushButton" name="MEF_Button">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>20</y>
            <width>50</width>
            <height>23</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="text">
           <string>MEF</string>
          </property>
         </widget>
         <widget class="QLabel" name="label">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>50</y>
            <width>70</width>
            <height>20</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Nb itérations :</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox">
          <property name="geometry">
           <rect>
            <x>40</x>
            <y>70</y>
            <width>50</width>
            <height>20</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox_3">
         <property name="maximumSize">
          <size>
           <width>16777215</width>
           <height>200</height>
          </size>
         </property>
         <property name="title">
          <string>contractions</string>
         </property>
         <widget class="QPushButton" name="static_MEF_with_contractions_Button">
          <property name="geometry">
           <rect>
            <x>30</x>
            <y>20</y>
            <width>75</width>
            <height>23</height>
           </rect>
          </property>
          <property name="text">
           <string>MEF statique</string>
          </property>
         </widget>
         <widget class="QSpinBox" name="spinBox_contraction_nb_iterations">
          <property name="geometry">
           <rect>
            <x>90</x>
            <y>50</y>
            <width>50</width>
            <height>20</height>
           </rect>
          </property>
          <property name="maximumSize">
           <size>
            <width>50</width>
            <height>16777215</height>
           </size>
          </property>
          <property name="alignment">
           <set>Qt::AlignCenter</set>
          </property>
          <property name="value">
           <number>1</number>
          </property>
         </widget>
         <widget class="QLabel" name="label_4">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>50</y>
            <width>70</width>
            <height>20</height>
           </rect>
          </property>
          <property name="sizePolicy">
           <sizepolicy hsizetype="MinimumExpanding" vsizetype="Preferred">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="maximumSize">
           <size>
            <width>70</width>
            <height>20</height>
           </size>
          </property>
          <property name="text">
           <string>Nb itérations :</string>
          </property>
         </widget>
         <widget class="QDoubleSpinBox" name="doubleSpinBox_gradient">
          <property name="geometry">
           <rect>
            <x>80</x>
            <y>110</y>
            <width>62</width>
            <height>22</height>
           </rect>
          </property>
          <property name="mouseTracking">
           <bool>true</bool>
          </property>
          <property name="toolTip">
           <string comment="puissance du gradiant de probabilité de contraction à partir des bords latéraux du tissu" extracomment="puissance du gradiant de probabilité de contraction à partir des bords latéraux du tissu">&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;puissance du gradient de probabilité de contraction à partir des bords latéraux du tissu.&lt;/p&gt;&lt;p&gt;ex:&lt;/p&gt;&lt;p&gt;0 : gradient uniforme&lt;/p&gt;&lt;p&gt;0.5 : gradient en 1/racine(x)&lt;/p&gt;&lt;p&gt;1 : gradient en 1/x&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_5">
          <property name="geometry">
           <rect>
            <x>20</x>
            <y>90</y>
            <width>121</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Puissance du gradient :</string>
          </property>
         </widget>
         <widget class="QLabel" name="label_6">
          <property name="geometry">
           <rect>
            <x>70</x>
            <y>110</y>
            <width>21</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>-</string>
          </property>
         </widget>
         <widget class="QLCDNumber" name="lcdNumber_proba">
          <property name="geometry">
           <rect>
            <x>83</x>
            <y>150</y>
            <width>61</width>
            <height>23</height>
           </rect>
          </property>
          <property name="value" stdset="0">
           <double>80.000000000000000</double>
          </property>
          <property name="intValue" stdset="0">
           <number>80</number>
          </property>
         </widget>
         <widget class="QLabel" name="label_7">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>150</y>
            <width>61</width>
            <height>16</height>
           </rect>
          </property>
          <property name="text">
           <string>Proba max :</string>
          </property>
         </widget>
         <widget class="QSlider" name="horizontalSlider_proba">
          <property name="geometry">
           <rect>
            <x>10</x>
            <y>180</y>
            <width>131</width>
            <height>19</height>
           </rect>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>80</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
    <item>
     <widget class="QGraphicsView" name="graphicsView">
      <property name="autoFillBackground">
       <bool>false</bool>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>1507</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>MEF_Button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>next_MEF()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>107</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>7</x>
     <y>819</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>next_MEF_without_area_constraint()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>64</x>
     <y>343</y>
    </hint>
    <hint type="destinationlabel">
     <x>2</x>
     <y>692</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>static_MEF_with_contractions_Button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>next_static_MEF_with_contractions()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>87</x>
     <y>688</y>
    </hint>
    <hint type="destinationlabel">
     <x>-1</x>
     <y>619</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>horizontalSlider_proba</sender>
   <signal>valueChanged(int)</signal>
   <receiver>lcdNumber_proba</receiver>
   <slot>display(int)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>88</x>
     <y>842</y>
    </hint>
    <hint type="destinationlabel">
     <x>134</x>
     <y>821</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>Tissue_Button</sender>
   <signal>clicked()</signal>
   <receiver>MainWindow</receiver>
   <slot>generate_tissue()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>82</x>
     <y>226</y>
    </hint>
    <hint type="destinationlabel">
     <x>0</x>
     <y>279</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>next_MEF()</slot>
  <slot>next_MEF_without_area_constraint()</slot>
  <slot>next_static_MEF_with_contractions()</slot>
  <slot>generate_tissue()</slot>
 </slots>
</ui>
